### PR TITLE
Update info about licenses

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9133

### CODE REVIEW

File: windows/deployment/windows-10-subscription-activation.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ If you’re running Windows 7, it can be more work.  A wipe-and-load approach w
 The following policies apply to acquisition and renewal of licenses on devices:
 - Devices that have been upgraded will attempt to renew licenses about every 30 days, and must be connected to the Internet to successfully acquire or renew a license.
 - If a device is disconnected from the Internet until its current subscription expires, the operating system will revert to Windows 10 Pro or Windows 10 Pro Education. As soon as the device is connected to the Internet again, the license will automatically renew.
-- Up to five devices can be upgraded for each user license.
+- Up to five devices can be upgraded for each user license. If user license is used for the sixth device, the operating system on the computer to which user has not logged in the longest will revert to Windows 10 Pro or Windows 10 Pro Education.
```
Comment (illfated) [inline windows/deployment/windows-10-subscription-activation.md:218]:
```suggestion
- Up to five devices can be upgraded for each user license. If the user license is used for a sixth device, the operating system on the computer to which a user has not logged in the longest will revert to Windows 10 Pro or Windows 10 Pro Education.
```
Somewhat long and difficult sentence, not quite sure how to make it easier to read and also understand, although it is fairly understandable as is (with adjustments like the one I have suggested).

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @illfated's suggestions.

**Comment by nam31** [comment]
@greg-lindsay PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by illfated** [comment]
Please note that we are still waiting for the suggested sentence adjustment is still waiting to be committed.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**